packages/auth/package.json

```diff
@@ -20,7 +20,7 @@
     "build:esm": "BABEL_ENV=ESM npx babel  --source-maps --config-file ../../bundle/babel.config.js --out-dir dist/esm --extensions '.ts' ./src",
     "build:cjs": "BABEL_ENV=COMMONJS npx babel  --source-maps --config-file ../../bundle/babel.config.js --out-dir dist/cjs --extensions '.ts' ./src",
     "build:umd": "webpack --config webpack.config.js",
-    "build:watch": "webpack --config webpack.config.js --watch",
```

Comment:
Nice! Maybe we can point this PR to https://github.com/CartoDB/toolkit/pull/78 branch, so we deal with watch there and merge this after that one

Reply:
Done!